jenkins-agents/stock/configure-jenkins.sh

```diff
@@ -18,7 +18,7 @@ apt install -y openjdk-8-jre-headless
 pip install docker
 
 #echo new cron into cron file
-echo '0 * * * * docker system prune --volumes --force --filter "until=1h"' | crontab
+echo '0 * * * * docker system prune --all --force --filter "until=1h"' | crontab
 
 docker network create jenkin
```

Comment:
is this supposed to say `jenkins`

Reply:
No--this docker network is used by the head node exclusively (and it's provisioned in ansible too, so we might be able to move it out of here). The name is a holdover from the old Jenkins head node.